src/main/java/cubicchunks/asm/mixin/core/client/MixinWorldClient.java

```diff
@@ -47,15 +47,12 @@
 	@Shadow public abstract boolean invalidateRegionAndSetBlock(BlockPos pos, IBlockState blockState);
 
 	@Override public void initCubicWorld() {
+		super.initCubicWorld();
```

Comment:
I had a few issues with some special cases with mixin in the past so now I'm overly cautious. (for example array initializers don't work in mixin classes).